scalafix-rules/src/main/scala/scalafix/internal/rule/RemoveUnused.scala

```diff
@@ -27,15 +27,18 @@ class RemoveUnused(config: RemoveUnusedConfig)
   override def isRewrite: Boolean = true
 
   private def warnUnusedPrefix = List("-Wunused", "-Ywarn-unused")
+  private def warnUnusedString = List("-Xlint", "-Xlint:unused")
   override def withConfiguration(config: Configuration): Configured[Rule] = {
     val hasWarnUnused = config.scalacOptions.exists(option =>
-      warnUnusedPrefix.exists(prefix => option.startsWith(prefix))
+      warnUnusedPrefix.exists(prefix =>
+        option.startsWith(prefix) || warnUnusedString.contains(option)
+      )
     )
     if (!hasWarnUnused) {
       Configured.error(
         s"""|The Scala compiler option "-Ywarn-unused" is required to use RemoveUnused.
             |To fix this problem, update your build to use at least one Scala compiler
-            |option that starts with -Ywarn-unused or -Wunused (2.13 only)""".stripMargin
+            |option like -Ywarn-unused, -Xlint:unused, or -Wunused (2.13 only).""".stripMargin
```

Comment:
I didn't add `-Xlint` to suggestions in the error message since `-Xlint` is for multi purposes.